docs/design/autoware-interfaces/ad-api/list/api/fail_safe/state.md

```diff
@@ -0,0 +1,15 @@
+# /api/fail_safe/state
+
+- Method: notification
+- Type: [autoware_ad_api_msgs/msg/FailSafeState](../../../types/autoware_ad_api_msgs/msg/fail_safe_state.md)
+
+## Description
+
+Get the fail safe state. For details, see the [fail safe state](./index.md).
+
+## Message
+
+| Name          | Type   | Description                                         |
+| ------------- | ------ | --------------------------------------------------- |
+| state         | uint16 | A value of the [fail safe state](./index.md).       |
```

Comment:
[Question] How can we know the severity of the failure? I guess you'll add another message for that.

Reply:
I plan to add MRC such as stop, slow down and move to the shoulder. But first, let's assume that there is only one MRC.